functions/database/health-checks/put-health-check.dynamodb.js

```diff
@@ -0,0 +1,18 @@
+const uuid = require('uuid/v4')
+const AWS = require('aws-sdk')
+const db = new AWS.DynamoDB.DocumentClient()
+
+module.exports = async (team) => {
```

Comment:
not needed brackets ()

Reply:
I would allow the brackets. It looks not intuitive for me in this case without. In case of short transformations as:
```
items.map(item => item.id)
```
I like version without additional brackets. But `async team => {` seems weird.

Reply:
I know.. this is matter of consistency and preferences.